queries.sql

```diff
@@ -8,3 +8,29 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg >= 10.50;
 SELECT * FROM animals WHERE neutered = true; 
 SELECT * From animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+UPDATE animals
+SET species = 'digimon' 
+WHERE name LIKE '%mon%';
+
+UPDATE animals
+SET species = 'pokemon' 
+WHERE name NOT LIKE '%mon%';
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT SP1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT SP1;
```

Comment:
After you `ROLLBACK TO SAVEPOINT SP1;` you must "update all animals' weights that are negative to be their weight multiplied by -1." as per project requirements.
![image](https://user-images.githubusercontent.com/38088818/171263814-f0abf5d1-d648-4132-92ca-fc945b2de001.png)

For example:
```
    UPDATE animals SET weight_kg = weight_kg * -1; 
    ROLLBACK TO SP1;
    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
```